From cca47f5c5364958b836270fbd640ac4062151174 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 12 May 2020 19:25:46 +0300
Subject: [PATCH] [SW-0] ppboot: specify preboot.h not local

The preboot.h file resides in include/ directory, so switch to use
global include instead of local directory one.

Change-Id: Iaf0ce0f2649c4ad0f55bf0d4d41299f52c4f0228
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/ppboot.c b/app/ppboot.c
index fd9edee..d45a216 100644
--- a/app/ppboot.c
+++ b/app/ppboot.c
@@ -10,10 +10,10 @@
 #include <cpu_ca53_cfg_regs.h>
 #include <hl_fw_if.h>
 #include <hl_boot_if.h>
+#include <preboot.h>
 #include <boot_desc.h>
 
 #include "common.h"
-#include "preboot.h"
 #include "uart.h"
 #include "spi.h"
 #include "pll.h"
